services/proxy-service.ts

```diff
@@ -27,6 +28,24 @@ export class ProxyService implements IProxyService {
 		this.$credentialsService.clearCredentials(this.credentialsKey);
 	}
 
+	public async getInfo(): Promise<string> {
```

Comment:
The object you are talking about which you'd like to return to fusion can be obtained through `getCache` method.
I can rename the `getInfo` to `getStringRepresentation` for example if the name troubles you but I see no point in this method returning an object because `getCache` does exactly that.
Moreover, if it returns an object I'd have to duplicate the printing logic (i.e. printing credentials if necessary)